react-hs-examples/redux/src/Base.hs

```diff
@@ -2,7 +2,7 @@ module Base (Lens, Ref((:-:))) where
 
 import React.Flux (SomeStoreAction)
 
--- Temporary and extremely evil
```

Comment:
Thanks :-)!  (And I really should stop mumbling to myself when doing reviews.  People take it seriously.  Sorry!)